tsl/test/expected/cagg_union_view-15.out

```diff
@@ -577,32 +591,39 @@ ORDER by 1;
 
 -- plan output
 :PREFIX SELECT * FROM mat_m1 ORDER BY 1;
-                                                               QUERY PLAN                                                                
------------------------------------------------------------------------------------------------------------------------------------------
+                                                                  QUERY PLAN                                                                   
+-----------------------------------------------------------------------------------------------------------------------------------------------
  Sort (actual rows=3 loops=1)
    Sort Key: _hyper_9_15_chunk.time_bucket
    Sort Method: quicksort 
    ->  Append (actual rows=3 loops=1)
          ->  Index Scan using _hyper_9_15_chunk__materialized_hypertable_9_time_bucket_idx on _hyper_9_15_chunk (actual rows=1 loops=1)
                Index Cond: (time_bucket < 25)
-         ->  HashAggregate (actual rows=2 loops=1)
-               Group Key: time_bucket(5, _hyper_7_11_chunk.a)
+         ->  Finalize GroupAggregate (actual rows=2 loops=1)
+               Group Key: (time_bucket(5, _hyper_7_11_chunk.a))
                Filter: ((sum(_hyper_7_11_chunk.c) > 50) AND ((avg(_hyper_7_11_chunk.b))::integer > 12))
-               Batches: 1 
                Rows Removed by Filter: 1
-               ->  Result (actual rows=6 loops=1)
-                     ->  Append (actual rows=6 loops=1)
-                           ->  Index Scan Backward using _hyper_7_11_chunk_ht_intdata_a_idx on _hyper_7_11_chunk (actual rows=2 loops=1)
-                                 Index Cond: (a >= 25)
-                                 Filter: ((b < 16) AND (c > 20))
-                           ->  Index Scan Backward using _hyper_7_13_chunk_ht_intdata_a_idx on _hyper_7_13_chunk (actual rows=3 loops=1)
-                                 Index Cond: (a >= 25)
-                                 Filter: ((b < 16) AND (c > 20))
-                           ->  Index Scan Backward using _hyper_7_14_chunk_ht_intdata_a_idx on _hyper_7_14_chunk (actual rows=1 loops=1)
-                                 Index Cond: (a >= 25)
-                                 Filter: ((b < 16) AND (c > 20))
-                                 Rows Removed by Filter: 2
-(23 rows)
+               ->  Sort (actual rows=3 loops=1)
+                     Sort Key: (time_bucket(5, _hyper_7_11_chunk.a))
+                     Sort Method: quicksort 
+                     ->  Append (actual rows=3 loops=1)
```

Comment:
Can this be ordered append and get rid of the sort node? Just wondering, let me know if this is a non-sense question :smile: 

And if not, why not?